swimster-api/routes/reservations.js

```diff
@@ -55,4 +55,15 @@ router.post('/listings/:listingId', requiresAuth, async (req, res, next) => {
     }
 })
 
+router.put('/listings/:listingId', requiresAuth, async (req, res, next) => {
+    try {
+        // changes status of DRAFT reservation to CONFIRMED
```

Comment:
where exactly does it set it to DRAFT?

Reply:
talked to Roberto offline - the schema sets the value of status when a reservation is created is created (when they hit Reserve button)

Reply:
done!